packages/keystatic/src/config.tsx

```diff
@@ -187,6 +189,9 @@ export function collection<
         ? K & string
         : never;
     }[keyof Schema][];
+    defaultSort?: {
+      column?: keyof Schema;
```

Comment:
Caveat: this allows _any_ field from the schema to be used as the sort column, but I couldn't figure out how to tell TypeScript that I only want to allow the values passed in the sibling `columns` property 😄 

Kind of a YOLO doesn't matter thing probably. But we could also rename it to `field` if we wanted to be pedantic.